### PR TITLE
add kbd67mkiirgbv2 and tofu96 keyboard

### PR DESCRIPTION
**Pull request by moyi4681**

### CODE REVIEW

File: keyboards/kbdfans/tofu96/tofu96.c [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    matrix_init_user();
+    led_init_ports();
+}
+
+void led_init_ports(void) {
+    setPinOutput(B2);
+    writePinHigh(B2);
+    setPinOutput(B0);
+    writePinHigh(B0);
+	setPinOutput(B1);
```
Comment (drashna) [inline keyboards/kbdfans/tofu96/tofu96.c]:
```suggestion
    setPinOutput(B1);
```

File: keyboards/kbdfans/tofu96/tofu96.c [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    writePinHigh(B1);
+}
```
Comment (drashna) [inline keyboards/kbdfans/tofu96/tofu96.c]: You shouldn't need to set the pins high, here.  IIRC, the led_update function is called at the end of each loop, so, it should be automatically set

File: keyboards/kbdfans/tofu96/tofu96.c [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+void led_init_ports(void) {
+    setPinOutput(B2);
+    setPinOutput(B0);
+	setPinOutput(B1);
+}
+bool led_update_kb(led_t led_state) {
+    if(led_update_user(led_state)) {
+        writePin(B2, !led_state.caps_lock);
+        writePin(B0, !led_state.num_lock);
+        writePin(B1, !led_state.scroll_lock);
```
Comment (fauxpark) [inline keyboards/kbdfans/tofu96/tofu96.c]:
```suggestion
        writePin(B1, !led_state.scroll_lock);
```

File: keyboards/kbdfans/tofu96/rules.mk [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   ATmega32A    bootloadHID
+#   ATmega328P   USBasp
+BOOTLOADER = atmel-dfu
+
+# Build Options
+#   comment out to disable the options.
+#
+BOOTMAGIC_ENABLE = yes	# Virtual DIP switch configuration(+1000)
```
Comment (fauxpark) [inline keyboards/kbdfans/tofu96/rules.mk]:
```suggestion
BOOTMAGIC_ENABLE = yes	# Virtual DIP switch configuration
```

File: keyboards/kbdfans/tofu96/rules.mk [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+MOUSEKEY_ENABLE = yes	# Mouse keys(+4700)
```
Comment (fauxpark) [inline keyboards/kbdfans/tofu96/rules.mk]:
```suggestion
MOUSEKEY_ENABLE = yes	# Mouse keys
```

File: keyboards/kbdfans/tofu96/rules.mk [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+EXTRAKEY_ENABLE = yes	# Audio control and System control(+450)
```
Comment (fauxpark) [inline keyboards/kbdfans/tofu96/rules.mk]:
```suggestion
EXTRAKEY_ENABLE = yes	# Audio control and System control
```

File: keyboards/kbdfans/tofu96/rules.mk [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+CONSOLE_ENABLE = no	# Console for debug(+400)
```
Comment (fauxpark) [inline keyboards/kbdfans/tofu96/rules.mk]:
```suggestion
CONSOLE_ENABLE = no	# Console for debug
```

File: keyboards/kbdfans/tofu96/readme.md [hunk 8]
```diff
@@ -0,0 +1,13 @@
+# TOFU96
+
+![TOFU96](https://kbdfans.com/collections/group-buy/products/tofu96-mechanical-keyboard-diy-kit)
+
+Keyboard Maintainer: [DZTECH](https://github.com/moyi4681)
```
Comment (fauxpark) [inline keyboards/kbdfans/tofu96/readme.md]:
```suggestion
* Keyboard Maintainer: [DZTECH](https://github.com/moyi4681)
```

File: keyboards/kbdfans/tofu96/readme.md [hunk 9]
```diff
@@ -0,0 +1,13 @@
+# TOFU96
+
+![TOFU96](https://kbdfans.com/collections/group-buy/products/tofu96-mechanical-keyboard-diy-kit)
+
+Keyboard Maintainer: [DZTECH](https://github.com/moyi4681)  
+Hardware Supported: DZTECH
```
Comment (fauxpark) [inline keyboards/kbdfans/tofu96/readme.md]:
```suggestion
* Hardware Supported: DZTECH
```

File: keyboards/kbdfans/tofu96/readme.md [hunk 10]
```diff
@@ -0,0 +1,13 @@
+# TOFU96
+
+![TOFU96](https://kbdfans.com/collections/group-buy/products/tofu96-mechanical-keyboard-diy-kit)
+
+Keyboard Maintainer: [DZTECH](https://github.com/moyi4681)  
+Hardware Supported: DZTECH  
+Hardware Availability: [KBDFANS](https://kbdfans.cn)
```
Comment (fauxpark) [inline keyboards/kbdfans/tofu96/readme.md]:
```suggestion
* Hardware Availability: [KBDFANS](https://kbdfans.cn)
```

File: keyboards/kbdfans/tofu96/keymaps/default/keymap.c [hunk 11]
```diff
@@ -0,0 +1,88 @@
...
+ */
+
+  [_BL] = LAYOUT_default(
+    KC_ESC,  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  KC_DEL,  KC_PSCR,    KC_SLCK, KC_PAUS, KC_PGUP, KC_PGDN, \
+    KC_GRV,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,           KC_BSPC,    KC_NLCK, KC_PSLS, KC_PAST, KC_PMNS, \
+    KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC,          KC_BSLS,    KC_P7,   KC_P8,   KC_P9,   KC_PPLS, \
+    KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT,                   KC_ENT,     KC_P4,   KC_P5,   KC_P6,            \
+    KC_LSPO,          KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH,          KC_RSPC, KC_UP,      KC_P1,   KC_P2,   KC_P3,   KC_PENT, \
+    KC_LCTL, KC_LGUI, KC_LALT,                            KC_SPC,                             KC_RALT, KC_RCTL, MO(1),   KC_LEFT, KC_DOWN, KC_RGHT,    KC_P0,   KC_PDOT           \
```
Comment (fauxpark) [inline keyboards/kbdfans/tofu96/keymaps/default/keymap.c]: The backslashes at the end of these lines are not necessary. You can keep them if you want as it isn't hurting anything, but IMO it looks neater without them.

**Pull request closed without merging**